PublishTestResultScreenshotV0/index.ts

```diff
@@ -19,10 +19,27 @@ const PARAM_ORGANIZATION = "organization";
 
 let project = tl.getVariable("System.TeamProject");
 let testApi: ta.ITestApi
+let buildId = tl.getVariable("Build.BuildId");
 
 async function run() {
     try {
+        tl.setResult(tl.TaskResult.SucceededWithIssues, "THE CURRENT TASK IS BEING DEPRECATED ON JULY 18, 2024", false);
```

Comment:
Not 100% sure this will show up anywhere. `tl` import should have a `warning` method.

Reply:
Updated code to use tl.warning()